druid/druid.sh

```diff
@@ -0,0 +1,319 @@
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat conf/druid/coordinator/jvm.config | xargs` -cp "${DRUID_DIR}/conf/druid/_common:${DRUID_DIR}/conf/druid/_common/hadoop-xml:${DRUID_DIR}/conf/druid/coordinator:lib/*" io.druid.cli.Main server coordinator"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/overlord.service
+[Unit]
+Description=Overlord
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat conf/druid/overlord/jvm.config | xargs` -cp "${DRUID_DIR}/conf/druid/_common:${DRUID_DIR}/conf/druid/_common/hadoop-xml:${DRUID_DIR}/conf/druid/overlord:lib/*" io.druid.cli.Main server overlord"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/broker.service
+[Unit]
+Description=Broker
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat examples/conf/druid/broker/jvm.config | xargs` -cp "${DRUID_DIR}/examples/conf/druid/_common:${DRUID_DIR}/examples/conf/druid/_common/hadoop-xml:${DRUID_DIR}/examples/conf/druid/broker:lib/*" io.druid.cli.Main server broker"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/historical.service
+[Unit]
+Description=Historical
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat conf/druid/historical/jvm.config | xargs` -cp "${DRUID_DIR}/conf/druid/_common:${DRUID_DIR}/conf/druid/_common/hadoop-xml:${DRUID_DIR}/conf/druid/historical:lib/*" io.druid.cli.Main server historical"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/middleManager.service
+[Unit]
+Description=MiddleManager
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat examples/conf/druid/middleManager/jvm.config | xargs` -cp "${DRUID_DIR}/examples/conf/druid/_common:${DRUID_DIR}/examples/conf/druid/_common/hadoop-xml:${DRUID_DIR}/examples/conf/druid/middleManager:lib/*" io.druid.cli.Main server middleManager"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+}
+
+function start_druid(){
+  cd ${DRUID_DIR}
+  systemctl daemon-reload
+  systemctl enable coordinator overlord broker historical middleManager
+  systemctl start coordinator overlord broker middleManager historical
+
+}
```

Comment:
Can we add "druid-" prefix for these service names? also use all lower case for service names, prefer "middle-manager" over "middleManager".